frontend/src/metabase/dashboard/actions/theme.ts

```diff
@@ -3,4 +3,6 @@ import { createAction } from "@reduxjs/toolkit";
 import type { DisplayTheme } from "metabase/public/lib/types";
 
 export const SET_DISPLAY_THEME = "metabase/dashboard/SET_DISPLAY_THEME";
-export const setDisplayTheme = createAction<DisplayTheme>(SET_DISPLAY_THEME);
+export const setDisplayTheme = createAction<DisplayTheme | null>(
```

Comment:
`theme` might also be `null`, which should be taken as the default theme of `light`

Reply:
Thanks for catching this!